sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/SearchIndexClientImplBuilder.java

```diff
@@ -208,7 +208,7 @@ public SearchIndexClientImplBuilder addPolicy(HttpPipelinePolicy customPolicy) {
      */
     public SearchIndexClientImpl buildClient() {
         if (apiVersion == null) {
-            this.apiVersion = "2020-06-30";
+            this.apiVersion = "2020-06-30-Preview";
```

Comment:
Can we use the SearchServiceVersion.getLatest()

Reply:
The generated code in-lines this. It is a good question on whether code generation could leverage creating the service enum though.